app/pytest/test_linux.py

```
@@ -59,7 +59,7 @@ def test_native_sim_boots():
         os.chmod("./zswatch_native_sim_64.exe", 0o755)
         # Run your SDL app
         proc = subprocess.Popen(
-            ["sudo", "./zswatch_native_sim_64.exe", "--bt-dev=hci0"],
+            ["sudo", "./zswatch_native_sim_64.exe", "--bt-dev=hci1"],
```

Ruff (0.14.5)

Starting a process with a partial executable path

(S607)